### PR TITLE
Add compatibility mode for Quantopian algorithms

### PR DESCRIPTION
**Pull request by ttt733**
This pull request will add a compatability mode (as the new default) which handles data in ways more similar to the Quantopian platform. While 1-to-1 equivalency cannot be guaranteed due to backend differences, the goal of this mode is to make the transition as smooth as possible.

### CODE REVIEW

File: pylivetrader/__main__.py [hunk 1]
```diff
@@ -82,6 +82,13 @@ def algo_parameters(f):
             default='INFO',
             show_default=True,
             help='The minimum level of log to be written.'),
+        click.option(
+            '-q', '--quantopian-compatible',
+            default=True,
+            type=bool,
+            show_default=True,
+            help=('Set 0 if compatibility with the Quantopian platform is not '
```
Comment (umitanuki) [inline pylivetrader/__main__.py]: set 0 not false?

File: pylivetrader/__main__.py [hunk 2]
```diff
@@ -82,6 +82,13 @@ def algo_parameters(f):
             default='INFO',
             show_default=True,
             help='The minimum level of log to be written.'),
+        click.option(
+            '-q', '--quantopian-compatible',
```
Comment (umitanuki) [inline pylivetrader/__main__.py]: it's a bit vague name. too long but something like quantopian-compatible-pricefilling or something

File: tests/test_backend/test_alpaca.py [hunk 3]
```diff
@@ -323,124 +325,124 @@ def historic_agg_data(size, *args, **kwargs):
...
                             'v': 217942,
-                            'd': 1535636280000},
+                            't': 1535636280000},
                            {'o': 223.42,
                             'c': 223.0002,
                             'h': 223.44,
                             'l': 222.91,
                             'v': 301328,
-                            'd': 1535636340000}]})
+                            't': 1535636340000}]})
```
Comment (umitanuki) [inline tests/test_backend/test_alpaca.py]: You need to add tests around compat and non-compat behavior. Hard to know if it works as expected.

Comment (ttt733) [reply tests/test_backend/test_alpaca.py]: Added a little coverage

**Pull request merged**